services/workflows-service/scripts/seed.ts

```diff
@@ -138,6 +141,7 @@ async function seed(bcryptSalt: string | number) {
       lastName: faker.name.lastName(),
       password: await hash('agent3', bcryptSalt),
       roles: ['user'],
+      avatarUrl: faker.image.people(200, 200, true),
```

Comment:
Let's change on instance of avatarUrl to be `null` so we could see the fallback (Initials)